app/config/config.yml

```diff
@@ -229,6 +229,7 @@ sonata_doctrine_orm_admin:
         types:
             list:
                 array: admin/CRUD/list_array.html.twig
+                datetime: admin/CRUD/list_datetime.html.twig
```

Comment:
is this type really used?

Reply:
Yes, for all date representation in admin, it was added to change a format and show month in french.